etl.py

```diff
@@ -124,7 +124,7 @@ def main(input_path_questions, input_path_answers, input_path_tags, output):
     q_df, a_df, t_df = read_data(
         spark, input_path_questions, input_path_answers, input_path_tags)
 
-    q_df = q_df.drop('Title', 'Body', 'ClosedDate')
+    q_df = q_df.drop('Title', 'ClosedDate')
```

Comment:
instead of modifying this dataframe, can you instead create a new one called `nlp_df`?